src/ia-item-user-lists.ts

```diff
@@ -301,6 +301,7 @@ export class IaItemUserLists extends LitElement {
   static styles = css`
     :host {
       display: block;
+      font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
Q: does this exclude our font?  This should be inherited, and if it is not in this repo, let's place it in the demo page css

Reply:
will try and check this approach

Reply:
PR updated